Automattic/wp-calypso
# Add search record details to Jetpack Search product card

### Changes proposed in this Pull Request

This PR displays the number of search records in the Search product card for a selected site.

### Testing instructions

- Visit the _Plans_ page with the offer reset flow enabled
- Select the _All_ or _Performance_ type to display the Search product card
- Check that the record details match the capture below
- Check that the price is updated as you switch from a yearly term to a monthly term
- Check that the price in the Jetpack Search product card depends on the site search records. You can update the number of records in your WPcom sandbox by hardcoding the value returned by current_records in `wp-content/lib/jetpack-plans/search.php`.
- Check that the tooltip contains a link to https://jetpack.com/upgrade/search/

### Screenshots

#### Mockups
<img width="555" alt="Screen Shot 2020-09-02 at 5 07 11 PM" src="https://user-images.githubusercontent.com/1620183/92038392-4b49b400-ed41-11ea-9d58-0780daa50b98.png">

#### Implementation
<img width="426" alt="Screen Shot 2020-09-03 at 9 47 44 AM" src="https://user-images.githubusercontent.com/1620183/92132015-9d87e500-edd4-11ea-8fec-13ae7076597b.png">
<img width="424" alt="Screen Shot 2020-09-03 at 10 58 50 AM" src="https://user-images.githubusercontent.com/1620183/92132031-a2e52f80-edd4-11ea-98eb-a6993d97b984.png">
